client/layout/masterbar-item-notifications.jsx

```diff
@@ -0,0 +1,140 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import ReactDom from 'react-dom';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import MasterbarItem from './masterbar-item';
+import Notifications from 'notifications';
+import store from 'store';
+
+export default React.createClass( {
+	displayName: 'MasterbarItemNotifications',
+
+	propTypes: {
+		user: React.PropTypes.object,
+		isActive: React.PropTypes.bool,
+		className: React.PropTypes.string,
+		onClick: React.PropTypes.func,
+	},
+
+	getInitialState() {
+		let newNote = false;
+		let user;
+
+		if ( this.props.user ) {
```

Comment:
This may not be necessary, since `! this.props.user` implies logged-out at https://github.com/Automattic/wp-calypso/pull/631/files#diff-5272ae9108aaabe9f9e3ceacb45c2987R65